src/components/services/Services.tsx

```diff
@@ -90,10 +88,7 @@ export function Services(props: Props) {
   // Methods to call immediately after login:
   useAsyncEffect(
     async () => {
-      // Show the password reminder on mount if required:
-      if (needsPasswordCheck && !isMaestro()) {
-        await Airship.show(bridge => <PasswordReminderModal bridge={bridge} navigation={navigation} />)
-      } else if (account != null) {
+      if (account != null) {
```

Comment:
Delete `needsPasswordCheck` from the list below, and then delete the selector above, as this is no longer needed.